rotkehlchen/api/v1/schemas.py

```diff
@@ -4152,6 +4129,18 @@ def __init__(self, chain_aggregator: 'ChainsAggregator') -> None:
             counterparties={x.identifier for x in chain_aggregator.get_all_counterparties()},
         )
 
+    @validates_schema
+    def validate_schema(
+            self,
+            data: dict[str, Any],
+            **_kwargs: Any,
+    ) -> None:
+        if data.get('to_timestamp') is not None and data['from_timestamp'] > data['to_timestamp']:
+            raise ValidationError(
+                message='from_timestamp must be less than or equal to to_timestamp',
+                field_name='to_timestamp',
+            )
```

Comment:
```suggestion
        if data.get('to_timestamp') is not None:
            super().validate_schema()
```
also this above doesn't check `validate_order_by_schema`. So probably you need to call always super(DBOrderBySchema).validate_schema() and then the if for `TimestampRangeSchema`. Not sure about the exact syntax but hope that you get it

Reply:
So to sum up: since the name is different it doesn't overwrite the logic and it is fine